clusterloader2/testing/watch-list/config.yaml

```diff
@@ -0,0 +1,71 @@
+{{$testDuration := "5m"}}
+name: huge-secrets
+namespace:
+  number: 1
+  prefix: "huge-secrets"
+tuningSets:
+- name: Uniform10qps
+  qpsLoad:
+    qps: 10
+steps:
+- name: Create secrets
+  phases:
+  - namespaceRange:
+      min: 1
+      max: 1
+      basename: huge-secrets
+    replicasPerNamespace: 400
+    tuningSet: Uniform10qps
+    objectBundle:
+    - basename: huge-secret
+      objectTemplatePath: "secret.yaml"
+- name: Start measurements
+  measurements:
+    - Identifier: TestMetrics
+      Method: TestMetrics
+      Params:
+        action: start
+    - Identifier: WaitForControlledPodsRunning
+      Method: WaitForControlledPodsRunning
+      Params:
+        action: start
+        apiVersion: apps/v1
+        kind: Deployment
+        labelSelector: group = watch-list
+        operationTimeout: 60s
+- name: Start the secret informer
+  phases:
+    - namespaceRange:
+        min: 1
+        max: 1
+        basename: huge-secrets
+      replicasPerNamespace: 1
+      tuningSet: Uniform10qps
+      objectBundle:
+        - basename: huge-secret
+          objectTemplatePath: role.yaml
+        - basename: huge-secret
+          objectTemplatePath: roleBinding.yaml
+        - basename: huge-secret
+          objectTemplatePath: "deployment.yaml"
+          templateFillMap:
+            Replicas: 1
+            Duration: {{$testDuration}}
+- name: Wait for the secret informer pod to be running
+  measurements:
+    - Identifier: WaitForControlledPodsRunning
+      Method: WaitForControlledPodsRunning
+      Params:
+        action: gather
+- name: Waiting {{$testDuration}} before cancelling the test
```

Comment:
Given my comment above - if we would use Job - instead of sleeping, I would just wait for all pods to finish here (i.e. effectively the job completion).

Reply:
np, we can switch to a Job.